cfme/infrastructure/virtual_machines.py

```diff
@@ -1473,7 +1473,18 @@ def step(self, *args, **kwargs):
 
 @navigator.register(Vm, 'candu')
 class VmUtilization(CFMENavigateStep):
-    VIEW = VMUtilizationAllView
+    @property
+    def get_view(self):
```

Comment:
This should just be `def VIEW(self):` property method, there's no need to wrap the method here just to change the name.

When is `VIEW=None` a valid use case that shouldn't raise an exception?

Reply:
@mshriver: :+1: right, It's my bad as `CFMENavigateStep` has `VIEW=None` so I can overwrite it with the same name property.